src/ApiFacade.php

```diff
@@ -22,11 +25,36 @@ public function __construct(Client $client, string $dataDir)
         $this->dataDir = $dataDir;
     }
 
-    public function createSnapshotTable(Table $snapshotTable, array $header, string $primaryKey): void
-    {
+    public function createSnapshotTable(
+        Table $snapshotTable,
+        Pattern $pattern
+    ): void {
         try {
+            if ($this->client->tableExists($snapshotTable->getTableId())) {
+                return;
+            }
             $this->createBucketIfNotExists($snapshotTable);
-            $this->createTableIfNotExists($snapshotTable, $header, $primaryKey);
+
+            $features = $this->client->verifyToken()['owner']['features'];
+            if (count(array_intersect(self::NATIVE_TYPES_FEATURES, $features)) > 0) {
```

Comment:
tímhle si nejsem úplně jistý no, teď to založí typovanou tabulku pokaždý když je zapnutá native-types fičura (je jedno jestli stará/nová)... ničemu by to asi vadit nemělo